packages/overture-schema-validation/README.md

```diff
@@ -0,0 +1,340 @@
+# Overture Schema Validation
+
+This is a constraint-based validation library intended for use with Overture Maps Pydantic schemas.
+This package provides reusable validation constraints that enforce data quality and business rules
+across Overture Maps feature data.
+
+## Overview
+
+This package provides constraint-based validation utilities for Overture Maps Pydantic schemas,
+offering both field-level and model-level validation capabilities.
+
+## Benefits of Using Constraints
+
+- **Cleaner code**: Less boilerplate validation logic
+- **Consistency**: Standardized validation patterns
+- **Reusability**: Constraints can be composed and reused
+- **Better errors**: Consistent, detailed validation error messages
+- **JSON Schema integration**: Constraints enhance generated JSON schemas
+
+## Installation
+
+```bash
+pip install overture-schema-validation
+```
+
+## Quick Start
+
+```python
+from typing import Annotated, List
+from pydantic import Field
+from overture.schema.core import StrictBaseModel
+from overture.schema.validation import (
+    LanguageTagConstraint,
+    CountryCodeConstraint,
+    UniqueItemsConstraint,
+)
+
+class PlaceProperties(StrictBaseModel):
+    # String pattern validation
+    language: Annotated[str, LanguageTagConstraint()] = Field(
+        ..., description="IETF BCP-47 language tag"
```

Comment:
Since this is the first example in the list, it is probably worth showing how you actually build the full field. I initially thought `...` was some kind of fancy syntax

Reply:
The correct (equivalent, but clearer) syntax, where you only need to provide the type, is:

```python
class PlaceProperties(StrictBaseModel):
    # String pattern validation
    language: Annotated[str, LanguageTagConstraint(), Field(
        description="IETF BCP-47 language tag"
    )]
```

`LanguageTag` is a type alias for that whole thing, so in practice, it's `language: LanguageTag`, and if it's optional, `language: LanguageTag | None = None`.

`...` was also new syntax to me (it's valid!). It means "not implemented"; in Pydantic, as a value, it means "this field has no default, so it's required".

(READMEs all need an update pass.)